danPortfolio.html

```diff
@@ -0,0 +1,140 @@
+<html>
+
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+	<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<script src="https://kit.fontawesome.com/5139af5f3f.js" crossorigin="anonymous"></script>
+	<link rel="stylesheet" type="text/css" href="danPortfolio.css">
+</head>
+
+<body>
+	
+	<nav>
+		<div id="nav" class="container-for-nav-buttons">
+			<a href="#about" class="three-nav-buttons">about</a>
+			<a href="#portfolio" class="three-nav-buttons">portfolio</a>
+			<a href="#contact" class="three-nav-buttons">contact</a>
+		</div>
+		<a href="https://docs.google.com/document/d/1YEz5F2rVdXZ46FvgSmDrT_3srlaZusB8/edit" target="_blank" class="download-cv">download CV</a>
+	</nav>
+	
+	<section class="intro" id="section-with-windows-image">
+		<div class="container-for-image-and-text-in-intro">
+			<img id="windows-image" src="https://images.adsttc.com/media/images/61e5/71ac/bcfa/3101/659e/e10c/newsletter/bdh-40.jpg?1642426848" alt="windows" width=100%>
+			<article class="brief-intro-text">
+				<h1>Welcome to Dan's programming portfolio</h1>
+			</article>
+		</div>
+				<div class="arrow">
+					<a class="arrow-link" href="#about"><i class="fa fa-chevron-down" style="font-size:50px"></i></a>
```

Comment:
at a certain screen size this overlaps the text:
<img width="830" alt="image" src="https://user-images.githubusercontent.com/25844355/213683799-67823b37-830f-44be-8cbc-b504bbb97a7f.png">